napari/layers/points/points.py

```diff
@@ -232,15 +305,65 @@ def __init__(
         self._is_selecting = False
         self._clipboard = {}
 
-        self.edge_color = normalize_and_broadcast_colors(
-            len(self.data), self._current_edge_color
-        )
-        self.face_color = normalize_and_broadcast_colors(
-            len(self.data), self._current_face_color
-        )
+        # set the edge color properties
+        self._edge_color_property = ''
+        if self._is_color_mapped(edge_color):
+            self._edge_color_mode = ColorMode.CYCLE
+        else:
+            self._edge_color_mode = ColorMode.DIRECT
+        if edge_color_cycle is None:
+            self._edge_color_cycle = cycle(
+                np.array([[1, 1, 0, 1], [0, 1, 0, 1]])
+            )
+        elif isinstance(edge_color_cycle, cycle):
+            self._edge_color_cycle = edge_color_cycle
+        else:
+            self._edge_color_cycle = transform_color_cycle(
+                color_cycle=edge_color_cycle,
+                elem_name='edge_color_cycle',
+                default='white',
+            )
+        self._edge_colormap = get_colormap(edge_colormap)
+        if isinstance(edge_colormap, str):
+            self._edge_colormap_name = edge_colormap
+        else:
+            self._edge_colormap_name = 'unknown_colormap'
+        self._edge_contrast_limits = edge_contrast_limits
+        self.edge_color = edge_color
+
+        # set the face color properties
+        self._face_color_property = ''
+        if self._is_color_mapped(face_color):
+            self._face_color_mode = ColorMode.CYCLE
+        else:
+            self._face_color_mode = ColorMode.DIRECT
+        if face_color_cycle is None:
+            self._face_color_cycle = cycle(
+                np.array([[1, 1, 0, 1], [0, 1, 0, 1]])
+            )
```

Comment:
As above, I think this is a terrible default! Sorry. 😬

Reply:
Changed to magenta/green 😛